The GRIB2 encoder/decoder (g2clib/1.9.0) shared library is called libg2c.so.

--- ncl-6.6.2.orig/config/Project	2019-02-28 00:44:39.000000000 +0100
+++ ncl-6.6.2/config/Project	2024-07-14 09:45:14.840207974 +0200
@@ -302,7 +302,7 @@
 #endif
 
 #ifndef	GRIB2lib
-#define       GRIB2lib   -lgrib2c -ljasper -lpng -lz -ljpeg
+#define       GRIB2lib   -lg2c -ljasper -lpng -lz -ljpeg
 #endif	/* GRIB2lib */
 
 #ifndef	EEMDlib
